style.css

```diff
@@ -155,8 +161,9 @@
     justify-content: center;
     flex-wrap: wrap;
     gap: 5px;
-    padding-right: 15px;
+    width: 100%;
     margin-top: 10px;
+    margin-left: -20px;
```

Comment:
- [ ] It seems you were not able to center them doing this. The problem is that you have padding messing with this 😅 

.ul-items {
    display: flex;
    align-items: center;
    justify-content: center;
    flex-wrap: wrap;
    gap: 5px;
    width: 100%;
    margin-top: 10px;
    padding: 0 10px;
  }

Kindly remove margin-left: -20px (Is not good practice to use negative margins for this purpose)
Please overwrite the existing padding (the second value of padding should be adjusted as in Figma )

I think this should work just fine 🤓